source/api/plugins/writing-a-plugin.md

```diff
@@ -59,6 +59,11 @@ This configuration contains all of the values that get passed into the browser f
 
 {% url 'For a comprehensive list of all configuration values look here.' https://github.com/cypress-io/cypress/blob/master/packages/server/lib/config.coffee %}
 
+It also includes the following extra values that are not part of the standard configuration. Note that these values cannot be modified from the plugins file.
```

Comment:
I would probably call this out more. 

```suggestion
It also includes the following extra values that are not part of the standard configuration. 

{% note warning %}
These values are read only and cannot be modified from the plugins file.
{% endnote %}
```

Reply:
The only problem is that there are more lines below this that refer to all the values, so it might be confusing. We'd need to restructure more lines or at least make it clear in the warning that we're only referring to `configFile` and `projectRoot`.